### DISCUSSION TITLE
Lost funds

### DISCUSSION THREAD

**Discussion by rrusth** [blockscout/blockscout#9079] [Support]
A send a withdraw of my bybit acoount to metamask, but doesn't show in my account. Are my fund lost, because i didn't get any information about it?

**Comment by Mojmir-R** [comment]
This is a completely wrong place to ask this. No one here can help you with this issue and you are only risking scammers contacting you. You MUST contact ByBit, no one else can help you with this.

**Comment by CharleneeveRivera** [comment]
> Reach out to cryptorecoverycenter@outlook.com with all the evidence of transaction

**Comment by parna1296** [comment]
Some one transfer Sidra from my account.i want to refund, please tell me how is it possible?

**Reply by GimluCom** [reply]
Please look at the answer above. You should contact the chain or platform you were using.
Blockscout is an open-source explorer, we do not have any control over transactions or users funds.

**Comment by MuhammadRaheel0** [comment]
I mistenkly withdrawal worng address kindly fixed it
